hazelcast/src/main/java/com/hazelcast/internal/util/phonehome/MapInfoCollector.java

```diff
@@ -103,4 +105,18 @@ private long countMapWithNativeInMemoryFormat() {
         return mapConfigs.stream()
                 .filter(mapConfig -> mapConfig != null && mapConfig.getInMemoryFormat() == InMemoryFormat.NATIVE).count();
     }
+
+    private long MapPutLatency(Node node) {
+        return (long) mapConfigs.stream().map(mapConfig -> node.hazelcastInstance.getMap(mapConfig.getName()))
+                .filter(map -> map.getLocalMapStats().getPutOperationCount() != 0L)
+                .mapToDouble(map -> map.getLocalMapStats().getTotalPutLatency() / map.getLocalMapStats().getPutOperationCount())
+                .average().orElse(0);
+    }
+
+    private long MapGetLatency(Node node) {
+        return (long) mapConfigs.stream().map(mapConfig -> node.hazelcastInstance.getMap(mapConfig.getName()))
+                .filter(map -> map.getLocalMapStats().getGetOperationCount() != 0L)
+                .mapToDouble(map -> map.getLocalMapStats().getTotalGetLatency() / map.getLocalMapStats().getGetOperationCount())
```

Comment:
Also, it is unclear what is the measurement unit here. It seems it sends everything in milliseconds, which is just too granular. How does this formula handle rounding? Please add some unittests covering rounding. Also it would be good to have a test which covers the divide-by-zero case. Last, are you sure 0 is a good marker for the no-data case? The average can also be 0 in cases when the map accesses are always fast (and that can easily happen).